ansible/playbooks/roles/keycloak/tasks/preflight/apply.yml

```diff
@@ -0,0 +1,38 @@
+---
+- name: Keycloak checks
+  run_once: true
+  delegate_to: localhost
+  module_defaults:
+    assert:
+      quiet: true
+  block:
+    - name: Include vars from keycloak role
+      include_vars:
+        file: roles/keycloak/vars/main.yml
+        name: keycloak_vars
+
+    - name: Assert postgresql component
+      when: keycloak_vars.specification.database.epiphany_managed
+      block:
+        - name: Ensure postgresql is provided
+          assert:
+            that:
+              - groups.postgresql is defined
+              - groups.postgresql | count > 0
+            fail_msg: At least one postgresql component has to be defined
+
+        - name: Execute if Postgresql is not standalone instance
+          when: groups.postgresql | count > 1
+          block:
+            - name: Include vars from application role
```

Comment:
You import these vars more than once, can you import them in one place and use them globally in keycloak tasks?